core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -284,6 +284,10 @@ private TableProperties() {}
   public static final String WRITE_AUDIT_PUBLISH_ENABLED = "write.wap.enabled";
   public static final String WRITE_AUDIT_PUBLISH_ENABLED_DEFAULT = "false";
 
+  public static final String ROLLBACK_COMPACTION_ON_CONFLICTS_ENABLED =
```

Comment:
I think the name feels a bit awkward to me, first it's a write side feature, so I think it's better to make it start with `write.`. Secondly this is to rollback a REPLACE operation, so feels to me it's better to call it something like `write.rollback-rewrite-enabled`?